client/src/components/common/DrawerToggleButton/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './style.css';
+
+const DrawerToggleButton = ({ clicked, className }) => {
+  return (
+    <article className={className} onClick={clicked}>
+      <span className="first"></span>
+      <span className="second"></span>
```

Comment:
The three classes are too common, try to use BEM
[BEM Link](http://getbem.com/naming/)